### DISCUSSION TITLE
LF RAT

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#289] [General]
Author: @tdegeorge 
Reviewer: @AbbyANoble 

https://www.overleaf.com/read/dzwsnptyphws#5fc1d9

**Comment by AbbyANoble** [comment]
Question 1 - Put all the A's and B's in math mode.  The ordered pairs too. 

Also, I'd say "ordered pairs" not "coordinate pairs".

**Comment by AbbyANoble** [comment]
Question 1 -  the graph is very large! Maybe that's fine, but mine in EQ and EL  are much smaller. Maybe we should make a general size range for graphs? I also think it might help to have some of the values on the axis. 


And did you mean for answer choice D to be (-4,0)? 

It also might be helpful to have (0,-4) as a choice somewhere.

**Reply by tdegeorge** [reply]
Not gonna lie - I had Steven make the graph for me! :) 

How do I make it smaller and with values?

**Reply by tdegeorge** [reply]
Oh wait - I have yours....I'm going to see if I can make the graphs smaller....

**Reply by AbbyANoble** [reply]
Here's a snippet of mine, except I added a label in. You can change the size with the height command. Looks like I'm using the axis and pgfplots and Steven used regular tikz. Not sure if that matters. 🤷‍♀️ Probably not since these are standalone and don't have to play nice with GitHub.

```
       \begin{tikzpicture}
        \begin{axis}[
                height=5.3cm,
                axis lines=center,
                grid=both,
                grid style={line width=.05pt, draw=gray!30},
                minor tick num=4,
                xtick={-10,-5,...,10},
                ytick={-10,-5,...,10},
                xmin=-10, xmax=10,
                ymin=-10, ymax=10,
                ]
                \addplot[mark=*] coordinates {(-4,6)} node[above,pos=1] {$A$};;
        \end{axis}
        \end{tikzpicture}
```

**Reply by tdegeorge** [reply]
I tried using yours but it gave me an error message. I was able to make the graph smaller and centered though.

**Reply by siwelwerd** [reply]
The syntax for tikz is a little different than what you've done with the axis.  Usually I just do something like `\begin{tikzpicture}[scale=0.3]` and change the 0.3 until it's a size I like.  Here think of 0.3 as giving you 30% of the original size.

**Reply by tdegeorge** [reply]
I was able to make it smaller and centered, but I can't seem to get the values on the axes to show

**Reply by AbbyANoble** [reply]
I use the scale too on occasion, but that also makes the letters too small sometimes. I generally use the `begin{axis}` stuff because that's whatever the person I originally learned (copy/pasted) from did. And I know how to make functions with it.

**Reply by AbbyANoble** [reply]
> Looks like I'm using the axis and pgfplots and Steven used regular tikz. Not sure if that matters. 🤷‍♀️ Probably not since these are standalone and don't have to play nice with GitHub.

I'm referencing which we decide to use not mattering. Not in how they work together. That does matter to get certain things to work.

**Reply by tdegeorge** [reply]
Ok, updated the graph based on what Drew and Steven suggested in the Tikz discussion. It does seem a little crowded with the point A, but I wanted one on one of the axes...

**Comment by AbbyANoble** [comment]
Question 2 - ordered pairs again, not coordinate. And math mode for the points again.

**Comment by AbbyANoble** [comment]
Question 4 - math mode needed on answer choices

**Comment by AbbyANoble** [comment]
Question 5 - I'd say "which of the following statements is true?"

**Comment by AbbyANoble** [comment]
At line 104 you have "%sigma notation" in the code and it doesn't seem to go with anything.

Which also made me notice that the readiness objectives aren't listed with which problems they go with. I guess it's not necessary, but I thought it may be nice to have them in there so we remember why we put the question in. And can convert to check it better later if we decide to.

**Reply by tdegeorge** [reply]
Oh! I forgot to put the objectives in there. I'll add those in

**Comment by AbbyANoble** [comment]
Question 7 - I'd rewrite the answer as 5/6 x - 5/6 (but where the fractions look nice!) so it looks more like the rest and what we'd want them to write anyway.

You can also use \dfrac instead of \frac to make the fractions less smushed.  (Which I potentially overuse now that my eyes are getting older. 😂)

**Comment by AbbyANoble** [comment]
Question 8 - not sure where you got your distractors from, but 8 may be a good one. You get that if you subtract the 6 first then divide by 3.

**Comment by AbbyANoble** [comment]
Question 9 and 10 - you can see if you like the \dfrac on those too so they aren't so small. (Again, maybe just my old eyes that rebel!)
